app/aws/s3.py

```diff
@@ -257,7 +278,7 @@ def get_job_from_s3(service_id, job_id):
                 "SlowDown",
             ]:
                 current_app.logger.exception(
-                    f"Retrying job fetch {FILE_LOCATION_STRUCTURE.format(service_id, job_id)} retry_count={retries}",
+                    f"Retrying job fetch  retry_count={retries}",
```

Comment:
Are we missing a variable to include here so we know what we're retrying?  I'm not sure what it'd be, I wonder if it's something to key off of with the `obj` variable, such as a property or something?

```suggestion
                    f"Retrying job fetch {VARIABLE HERE} retry_count={retries}",
```

Reply:
Or maybe it's what you have below, `{service_id}`?

Reply:
fixed